nxc/connection.py

```diff
@@ -121,7 +121,10 @@ def __init__(self, args, db, host):
         try:
             self.proto_flow()
         except Exception as e:
-            self.logger.exception(f"Exception while calling proto_flow() on target {self.host}: {e}")
+            if "ERROR_DEPENDENT_SERVICES_RUNNING" in str(e):
+                self.logger.error(f"Exception while calling proto_flow() on target {self.host}: {e}")
```

Comment:
Should we output something more useful to the user, explaining why this is happening?

Reply:
Hmm I thought about it but didn't know what to add. The error message is not that bad imo. Maybe that it is probably the remote registry service or we add which service it was? Or we open up a wiki page explaining error codes? (But the latter is a bit redundant as you could just google it)

Reply:
I think this should only be thrown during `smbexec` since we create and execute a service, but it looks like it's being thrown during reg-winlogon, so I'll check that out... maybe it's trying to stop the remote registry service.

Reply:
Ah yeah so this is trying to stop the remote registry service, which other stuff can rely on. It's also importing code from the secretsdump example, instead of calling impacket code directly... which is not great IMO. I guess for now we can just leave it at this, since it's not a critical failure, but we should clean this up eventually.

Reply:
Hmm yeah but i think this is just because of impacket. RemoteOperations is (whyever) defined in secretsdump so you have to import it from there. Take a look at other modules, a lot import it from there.